.gitignore

```diff
@@ -4,3 +4,5 @@ __pycache__/
 # javascript
 node_modules
 package-lock.json
```

Comment:
I actually have a question here : why do you ignore the `package-lock.json` ?
The command `npm ci` (that is not very well known) reads this file instead of the `package.json` file. This helps to ensure a reproducible build by making sure everyone use the same dependencies. Looks like good stuff to me. Is there a drawback I don't see ?

Reply:
because I'm using yarn instead of npm. I need yarn for the yarn workspace feature